toolkits/golang_min_version.go

```diff
@@ -1,5 +1,5 @@
 package toolkits
 
 const (
-	minGoVersionForToolkit = "1.20.8"
+	minGoVersionForToolkit = "1.21.10"
```

Comment:
We update to this version because `slices` was introduced in Go 1.21 (based on: [this article](https://www.gopherguides.com/articles/golang-slices-package)) right?

Reply:
Yes and this version is available on all not self hosted stacks as well.